pkg/provider/provider.go

```diff
@@ -11,6 +11,16 @@ import (
 	"github.com/hewlettpackard/hpegl-provider-lib/pkg/registration"
 )
 
+const (
+	// IAMVersionGLCS is the IAM version for GLCS
+	IAMVersionGLCS = "glcs"
```

Comment:
Might be worth defining a custom string type for the IAM version and using that later in the code. This should give some useful linting hints in the future to ensure all switch statements are exhaustive and considering all possible IAM versions.

Reply:
Done